### PR TITLE
deployment + verification scripts for sFRAX

### PR DESCRIPTION
**Pull request by tbrent**
Needs 2 reviewers to sign off on

### CODE REVIEW

**Comment by tbrent** [comment]
(small thing)

I see no reason to code the `FORK_BLOCK` in the CI config; it should be the case that when someone runs a local devchain that they are able to run our deploy scripts. So whatever block number it uses, it should also work locally. Seems like `fork-block-number.default` is the most natural thing to use.

**Review by julianmrodri — APPROVED** [review]
LGTM checked oracle timeout and error for Mainnet

**Review by julianmrodri — APPROVED** [review]
LGTM

File: scripts/deployment/phase2-assets/collaterals/deploy_sfrax.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    {
+      priceTimeout: priceTimeout.toString(),
+      chainlinkFeed: networkConfig[chainId].chainlinkFeeds.FRAX,
+      oracleError: fp('0.01').toString(), // 1%
+      erc20: networkConfig[chainId].tokens.sFRAX,
+      maxTradeVolume: fp('1e6').toString(), // $1m,
+      oracleTimeout: oracleTimeout(chainId, '3600').toString(), // 1 hr
+      targetName: hre.ethers.utils.formatBytes32String('USD'),
+      defaultThreshold: fp('0.02').toString(), // 2% = 1% oracleError + 1% buffer
+      delayUntilDefault: bn('86400').toString(), // 24h
```
Comment (akshatmittal) [inline scripts/deployment/phase2-assets/collaterals/deploy_sfrax.ts:59]: Verified parameters! Surprised that the heartbeat for FRAX is 1hr!

**Review by akshatmittal — APPROVED** [review]

**Pull request merged**